Page2.html

```diff
@@ -0,0 +1,247 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<title>CSS Template</title>
+<meta charset="utf-8">
+<meta name="viewport" content="width=device-width, initial-scale=1">
+
+  <script src="https://kit.fontawesome.com/aa5632ef5d.js" crossorigin="anonymous"></script>
+
+
+
+</head>
+<body>
+  <a href="#index" class="hidden">skip to main content</a>
+
+
+  <header>
+    <link rel="stylesheet" href="Fonts.css">
+
+    <div class="gridwrapper_header">
+    <div class="head1-header">
+      <div class="img-with-text">
+          <img src="images/adoptadog-logo.svg" alt="Seattle’s premiere dog adoption agency">
+      <p>Seattle’s premiere dog adoption agency</p>
+  </div>
+</div>
+
+  <div class="head2-header">
+    <div class="footer1">
+            <img src="images/cart-icon.svg" alt="Seattle’s premiere dog adoption agency" width="30" height="30">
```

Comment:
Don't use hard coded values in the HTML for widths and heights, do that in the css